docs/user-guide/work-packages/set-change-dates/README.md

```diff
@@ -0,0 +1,138 @@
+| ------------------------------------------------------------ | --------------------------------------------------------- |
+| [Set start and finish dates](#set-start-and-finish-dates)    | How to set start and finish dates, and how to change them |
+| [Scheduling mode](#scheduling-mode)                          | Enabling manual scheduling and what it does               |
+| [Warning and information banners](#warning-and-information-banner) | What the different warning and information banners mean   |
+
+## Set start and finish dates
+
+You can change the start and finish dates of work packages opening the date picker. 
+
+![standard-date-picker](standard-date-picker.png)
+
+1. [Manual scheduling](#scheduling-mode) checkbox
+2. Start and finish date input fields
+3. Mini calendars for date range preview
+4. Save and cancel buttons
+
+You can open the date picker for a work package from a number of different places:
+
+- By clicking on the date field in the work package [details view](../../work-packages/work-package-views/#work-package-full-screen-view)
+- By clicking the the date field in the work pacakge [split screen view](../../work-packages/work-package-views/#work-package-split-screen-view) (from any other view, including [notification center](../../notifications), [team planner](../../team-planner/), [boards](../../agile-boards)...)
+- By clicking the start or finish date fields in [list view](../../work-packages/work-package-views/#work-package-list-view)
+
+
+### Using the date picker
+
+You can enter dates either by typing them in into the start and finish date fields or by clicking on individual dates in the mini calendar below.
+
+> **Note**: Typed dates need to be in the standard ISO 8601 format: **YYYY-MM-DD** (so 12 October 2022 would be 2022-10-12). 
+>
+> They will nevertheless be displayed in your desired format (usually depending by your region or language).
+
+You can also click on the **Today** link below the start and finish date fields to use today's date.
+
+A more intuitive way to select start and end dates is to simply click on two different dates in the mini calendars below. Two calendar months are displayed to give you more visibility. 
+
+Start by clicking on a start date. This will enter the selected date as the start date,  mark it with a dark color on the calendar and move the focus automatically to the finish date. Hovering on different finish dates will give you a preview of which dates are  included if you click. Once you have decided on a finish date, click on it to select it. This will enter the finish date in the date field and mark the end date with another dark color.
+
+ To confirm the selected dates, click on the **Save** button. The green message on top of the work package indicates a successful update.
+
+![successful-update](successful-update.png)
+
+To clear a date field, simply click on the **clear icon** ("×") icon when the field is in focus.
+
+![clear-date-field](clear-date-field.png)
```

Comment:
Just an FYI: please use full english text in the [image descriptions](https://www.openproject.org/docs/development/contribution-documentation/documentation-style-guide/#add-the-image-link-to-content), since they are read aloud by screenreaders, the dashes don't do good in most applications.

Reply:
Ok, added full descriptions. I was going using existing documentation as a guide, so some existing pages need to be updated too it looks like.